libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -423,3 +424,26 @@ export function disableComputeRatio<T extends IAttributeOrMeasure>(item: T): T {
 
     return item;
 }
+
+/**
+ * Describes the type of the function used to modify the bucket items.
+ *
+ * @public
+ */
+export type BucketItemModifier = (bucketItem: IAttributeOrMeasure) => IAttributeOrMeasure;
+
+/**
+ * Creates a new bucket with modified bucket items (retrieved by applying the modifyFunction to the bucketItem).
+ *
+ * @param bucket - maybe a simple measure where compute ratio should be disabled
```

Comment:
this param description does not look valid :)

Reply:
Thanks, I've updated the PR.